addons/apecloud-postgresql/templates/configconstraint.yaml

```diff
@@ -2,11 +2,29 @@
 apiVersion: apps.kubeblocks.io/v1beta1
 kind: ConfigConstraint
 metadata:
-  name: apecloud-postgresql14-cc
+  name: {{ include "apecloud-postgresql14.configConstraint" . }}
   labels:
     {{- include "apecloud-postgresql.labels" . | nindent 4 }}
+  annotations:
+    {{- include "apecloud-postgresql.annotations" . | nindent 4 }}
 spec:
+  reloadAction:
+    shellTrigger:
+      sync: true
+      command:
+        - "update-parameter.sh"
+      scriptConfig:
+        scriptConfigMapRef: reload-tools-script
```

Comment:
For resource naming that involves multiple usages, it's best to define and reference them uniformly to avoid omissions during subsequent modifications.